props/saber_BC_buttons.h

```diff
@@ -1169,6 +1183,8 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
   bool min_vol_reached_ = false;
   bool sequential_quote_ = false;
   bool spam_blast_ = false;
+  bool speaking_ = false;
```

Comment:
I'm not sure i understand this variable.
Can we add a comment that explains what it's for?

Reply:
Comment added.
It's to avoid overlap of battery.wav playing from On-Demand battery level when doing Spoken Battery Level as they both invoke EFFECT_BATTERY LEVEL.